index.js

```diff
@@ -1,17 +1,35 @@
 
-var { benchBranch, benchmarkRuntime } = require("./bench");
+const { benchBranch, benchmarkRuntime } = require("./bench");
+
+let allowedUsers = process.env.ALLOWED_USERS || "";
+allowedUsers = allowedUsers.split(",").map((item) => Number(item.trim())).filter((item) => !Number.isNaN(item) && item !== 0);
+
+function isAllowed(senderId) {
+  return allowedUsers.includes(senderId);
```

Comment:
if `ALLOWED_USERS` not set it should allow anyone

Reply:
That's what I was thinking about too - And decided that I would add another special flag to allow any. And which would have to be explicitly set to allow any. Basically In order to prevent "misconfiguration" when allowed user is not set ( eg. forgotten ). 

Does this make sense ? or is ok to allow any when not set ?

Reply:
I see this just as an optional feature

Reply:
but i would explain this in the readme